<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Titlu">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Titlu:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Descriere">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Descriere</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Data">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Finalizat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Finalizat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Adaugă</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditDueDate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTitle">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDescription">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCompleted">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTasks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Șterge Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExport">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportă XML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonImport">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Importă XML</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStyle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Fusion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Windows</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Macintosh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonStergeTot">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Șterge Tot</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTasks">
    <property name="title">
     <string>Tasks</string>
    </property>
   </widget>
   <addaction name="menuTasks"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
